src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java

```diff
@@ -86,7 +86,7 @@ public String send(Letter letter, String serviceName) throws JsonProcessingExcep
             throw new SendMessageException("Could not send message to ServiceBus with " + messageId);
         }
 
-        return messageId;
+        return id.toString();
```

Comment:
have you tried to return UUID itself and check the response body? i think spring should automatically marshal it to string. unless i'm naively hoping for that

it would be better to show in documentation that response body is UUID, that's all. not just some string

Reply:
I will give it a try.Actually I  was returning UUID later I thought will return string as I was not sure whether spring will marshal it :)